poutyne/framework/callbacks/notification.py

```diff
@@ -0,0 +1,73 @@
+from typing import Dict, Union
```

Comment:
You should import this this in `poutyne/framework/callbacks/__init__.py` so that we can do `from poutyne import NotificationCallback`.

Reply:
Yes, I forgot.